### PR TITLE
set secure for all envs

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

File: cla_frontend/settings/base.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def get_env_value(var_name):
 # TODO change this ?
 #SESSION_ENGINE = 'django.contrib.sessions.backends.file'
 SESSION_ENGINE = 'django.contrib.sessions.backends.signed_cookies'
-SESSION_COOKIE_SECURE = os.environ.get('CLA_ENV', '') in ['prod', 'staging']
+SESSION_COOKIE_SECURE = bool(os.environ.get('CLA_ENV'))
```
Comment (kerin) [inline cla_frontend/settings/base.py]: Is that right?

```
>>> bool('False')
True
>>> bool('0')
True
```

Comment (s-block) [inline cla_frontend/settings/base.py]: if it exists then I want it to be true - i.e. for all envs except local

Comment (kerin) [inline cla_frontend/settings/base.py]: I'll wind my neck in then.

**Comment by s-block** [comment]
retest this please

**Pull request merged**